tests/commons.py

```diff
@@ -1,3 +1,8 @@
+#
+# Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+# or more contributor license agreements. Licensed under the Elastic License 2.0;
+# you may not use this file except in compliance with the Elastic License 2.0.
+#
```

Comment:
Huh. We probably need a test to ensure that this appears on all our files. 
Nice catch.

Reply:
I think we check it somewhere for _some_ files. Maybe the `tests` directory is ignored?